dist/10-10CG-schema.json

```diff
@@ -365,27 +365,17 @@
             "626"
           ]
         },
-        "lastTreatmentFacility": {
-          "type": "object",
-          "additionalProperties": false,
-          "required": [
-            "name",
-            "type"
-          ],
-          "properties": {
-            "name": {
-              "type": "string",
-              "minLength": 1,
-              "maxLength": 100
-            },
-            "type": {
-              "type": "string",
-              "enum": [
-                "hospital",
-                "clinic"
-              ]
-            }
-          }
+        "lastTreatmentFacilityName": {
```

Comment:
@kevinmirc what do you think about these names?

Reply:
The names look good to me.